homeassistant/components/mqtt/siren.py

```diff
@@ -279,8 +283,8 @@ def state_message_received(msg: ReceiveMessage) -> None:
                         invalid_siren_parameters,
                     )
                     return
+                self._extra_attributes = dict(self._extra_attributes)
```

Comment:
To be able to track changes to `self._extra_attributes` we assign an immutable copy.

Reply:
Please add a comment for that since someone will likely try to optimize this away in the future if we don't

Reply:
done